secretmanager/api/SecretManager.Samples/CreateUserManagedReplicationSecret.cs

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.SecretManager.V1;
+
```

Comment:
This sample does not have a region tag. Where in docs is it going to be used?
We don't usually accept samples that are not going to be used in docs.

Reply:
Hi Amanda, please check b/222386112. As part of this bug it was needed to add sample code for user managed replication mainly for not strongly typed languages but we decided to add it for all of the languages. There is no plan to add the samples in docs as of now thats why region tag is not needed.

Reply:
I've left a comment on the issue. Thanks for the context.